deducing-this.md

```diff
@@ -366,15 +391,20 @@ It also works as a more obvious way to teach how `std::bind` and `std::function`
 
 ## ABI implications for `std::function` and related
 
-If references and pointers do not have the same representation for member functions, this effectively says "for the purposes of the `this`-designated first parameter, they do."
+The representation of references is not defined. This leads to the following
+constraint: if references and pointers do not have the same representation for member functions, this effectively says "for the purposes of the `this`-designated first parameter, they do."
```

Comment:
Extra new-line between following and constraint?

Reply:
Done.